### DISCUSSION TITLE
Support regex search for email templates in dev server command

### DISCUSSION THREAD

**Discussion by midanosi** [resend/react-email#1350] [Ideas]
### Goals

1. Regex search for template files within a specified folder

### Background

Scenario:
I have a react-email template file, called `SendWeeklyDigest.tsx`. I have another file called `sendWeeklyDigest.ts`, which fetches some data, generates the email body via the template, then sends the email.

I would like to group my concerns by function execution. So those two files both live in a folder `digests`. This isn't 

This isn't possible at the moment, as the only way to store react-email template files in a repository (as I understand it), is by having one single folder that is only allowed to contain react-email templates.

### Proposal

Add `--include` option to dev server command that allows specifying where my react-email templates live.

And then suggesting files as templates, e.g. `SendWeeklyDigest.template.tsx` or `SendWeeklyDigest.template.tsx`

Then the dev server command would be `email dev --include src/email/*.template.tsx --port 3003`

**Comment by gabrielmfern** [comment]
We did already think of this kind of thing and have a solution implemented for it, just haven't quite got around to documenting it.
Here's a part from a PR I'm writing that touches a bit on this:

https://github.com/resend/react-email/blob/7cbf829f5cb2e6f8f2aaf3fad51e4bb186fafb29/apps/docs/cli.mdx#L86-L103
